infer/environment.cc

```diff
@@ -397,6 +397,7 @@ void Environment::updateKnowledge(core::Context ctx, core::LocalVariable local,
     if (send->args.empty()) {
         return;
     }
+    // TODO(jez) We should probably update this for whatever lazyResolve thing we pick
```

Comment:
I'm thinking about leaving this TODO here and coming back to it if/when we realized that it's blocking something.